### PR TITLE
Extended simulation of neutron  background

### PR DESCRIPTION
**Pull request by civanch**
By request of muon upgrade TDR, thermal neutron simulation extended two another Physics List used for this study. Time window extended from 0.1 second to 10 second.

Added extra header which will be used soon.

Should not affect mainstream simulation.

### CODE REVIEW

**Comment by civanch** [comment]
please test

**Comment by civanch** [comment]
+1

File: SimG4Core/Application/python/NeutronBGforMuonsXS_Term_cff.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import FWCore.ParameterSet.Config as cms
+
```
Comment (davidlange6) [inline SimG4Core/Application/python/NeutronBGforMuonsXS_Term_cff.py]: Hi @civanch - please add some documentation of whats happening in this customize function and how it differs from the others. 

Second, could it not just rely on NeutronBGforMuonsXS_cff.py for easier future maintenance?

**Comment by davidlange6** [comment]
-1
small comments are to be addressed by @civanch

**Comment by civanch** [comment]
@davidlange6 , comments are added to custom fragments. The common part is taken out into a common fragment. Unfortunately, by mistake updated cavern PR was peaked up here, so this PR will be closed and #17833 is already opened.

**Pull request closed without merging**